### ISSUE TITLE
dev.isar.isar_flutter_libs.IsarInitializer.create

### ISSUE THREAD

**Issue by patilvishwanath** [isar/isar#1159]
### Steps to Reproduce

App crashing at startup & throwing runtime exception

### Code sample

```dart
class IsarStorage {
  IsarStorage._();
  static final IsarStorage isarStorage = IsarStorage._();


  late Isar isar;

  Future<void> init() async {
    isar = await Isar.open([CacheItemSchema]);
  }


 Future<void> delete(String key) async {
    await isar.writeTxn(() async {
      final success = await isar.cacheItems.filter().keyStartsWith(key).deleteFirst();
      print('Cache --  Isar deleted for  ${key} : $success');
    });
  }

  Future<CacheItem?> getData<T>(String key) async {
    final success =  await isar.cacheItems.
  filter().keyMatches(key).findFirst();
    print('Cache -- Isar getData for param ${key} : ${success?.key}');
    return success;
  }

  

  Future<void> set<T>(CacheItem item) async {
    await isar.writeTxn(() async {
      final success = await isar.cacheItems.put(item);
      print('Cache -- Isar put key ${item.key} : $success');
    });
  }
}

```
main.dart
```dart
await IsarStorage.isarStorage.init();
```

### Details

Platform: samsung a22x (Galaxy A22 5G)- Android 11, vivo 2127  Android 13, 
vivo 2135 Android 12
Flutter version: 3.3.10
Isar version: 3.0.0

<details><summary>Log</summary>
<p>

Exception java.lang.RuntimeException: Unable to get provider androidx.startup.InitializationProvider: androidx.startup.StartupException: java.lang.UnsatisfiedLinkError: dlopen failed: library "libisar.so" not found
  at android.app.ActivityThread.installProvider (ActivityThread.java:8215)
  at android.app.ActivityThread.installContentProviders (ActivityThread.java:7746)
  at android.app.ActivityThread.handleBindApplication (ActivityThread.java:7566)
  at android.app.ActivityThread.access$1500 (ActivityThread.java:301)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2177)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:246)
  at android.app.ActivityThread.main (ActivityThread.java:8653)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:602)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1130)
Caused by androidx.startup.StartupException: java.lang.UnsatisfiedLinkError: dlopen failed: library "libisar.so" not found
  at androidx.startup.AppInitializer.doInitialize (AppInitializer.java:187)
  at androidx.startup.AppInitializer.discoverAndInitialize (AppInitializer.java:238)
  at androidx.startup.AppInitializer.discoverAndInitialize (AppInitializer.java:206)
  at androidx.startup.InitializationProvider.onCreate (InitializationProvider.java:45)
  at android.content.ContentProvider.attachInfo (ContentProvider.java:2429)
  at android.content.ContentProvider.attachInfo (ContentProvider.java:2399)
  at android.app.ActivityThread.installProvider (ActivityThread.java:8210)
Caused by java.lang.UnsatisfiedLinkError: dlopen failed: library "libisar.so" not found
  at java.lang.Runtime.loadLibrary0 (Runtime.java:1087)
  at java.lang.Runtime.loadLibrary0 (Runtime.java:1008)
  at java.lang.System.loadLibrary (System.java:1664)
  at dev.isar.isar_flutter_libs.IsarInitializer.create (IsarInitializer.java:16)
  at dev.isar.isar_flutter_libs.IsarInitializer.create (IsarInitializer.java:11)
  at androidx.startup.AppInitializer.doInitialize (AppInitializer.java:180)

</p>
</details>

---

 - [x] I searched for similar issues already
 - [x] I filled the details section with the exact device model and version
 - [x] I am able to provide a reproducible example

**Comment by JobiJoba** [comment]
Same issue as #1080  
(and some other duplicate)

**Comment by simc** [comment]
This is fixed in v3.1.0 which will be available soon on pub.dev.
